addon/ng2/commands/generate.ts

```diff
@@ -48,6 +91,32 @@ function mapBlueprintName(name) {
   return mappedName ? mappedName : name;
 }
 
+function importIntoAppModule(args, project, task) {
```

Comment:
If we're auto-importing stuff into NgModules, we should also:
- allow to user to bypass this functionality (they might have multiple NgModules in their app for instance)
- document it in our readme somewhere so it's not a surprise